assets/js/dashboard/api.ts

```diff
@@ -15,7 +15,7 @@ export class ApiError extends Error {
   }
 }
 
-function serialize(obj: Record<string, string | boolean | number>) {
+export function serialize(obj: Record<string, string | boolean | number>) {
```

Comment:
Given we now export this, the name is kind of hard to grok.

```suggestion
export function serializeUrlParams(params: Record<string, string | boolean | number>): string {
```